runescape-client/src/main/java/ObjectComposition.java

```diff
@@ -252,7 +252,7 @@
    @ObfuscatedSignature(
       signature = "Lgr;"
    )
-   IterableHashTable field3558;
+   IterableHashTable params;
```

Comment:
You're missing an @Export annotation. It's easier to see such things if you make 1 commit per mapping.